spec/consumer_spec.rb

```diff
@@ -1,11 +1,12 @@
 require_relative 'spec_helper'
 
 describe Consumer do
-  let(:message_values) { [:delivery_info1, :headers1, "message1_payload"] }
+  let(:yield_values) { [:delivery_info1, :headers1, "message1_payload"] }
```

Comment:
Can we do this in another way? I think I understand the difference between `yield_values` and `message_values`, but it confused me at first. Maybe it's easier if we inline the values instead of the `let` (it's only used once anyway)?